src/lib/toolchain_test.rs

```diff
@@ -39,8 +49,8 @@ fn wrap_command_with_args() {
     let args = output.args.unwrap();
     assert_eq!(args.len(), 5);
     assert_eq!(args[0], "run".to_string());
-    assert_eq!(args[1], "mychain".to_string());
-    assert_eq!(args[2], "testcommand".to_string());
+    assert_eq!(args[1], "nightly".to_string());
```

Comment:
this needs to change to channel for tests to pass

Reply:
Oof, sorry for that, just pushed a fix :)